metaci/release/utils.py

```diff
@@ -84,9 +111,84 @@ def send_release_webhook(project_config, release):
     result = response.json()
     if result["success"]:
         with transaction.atomic():
+            if settings.METACI_START_STOP_WEBHOOK:
+                for i in range(0, len(implementation_steps) - 1):
+                    implementation_steps[i].external_id = result["implementationSteps"][
+                        i
+                    ]
             case_id = result["id"]
             case_url = settings.METACI_CHANGE_CASE_URL_TEMPLATE.format(case_id=case_id)
             release.change_case_link = case_url
             release.save()
     else:
         raise Exception("\n".join(err["message"] for err in result["errors"]))
+
+
+def send_start_webhook(project_config, release, role, config_item):
+    if (
+        release is None
+        or not settings.METACI_RELEASE_WEBHOOK_URL
+        or not settings.METACI_START_STOP_WEBHOOK
+    ):
+        return
+    if not config_item:
+        raise Exception(
+            "Error sending start webhook, please include a configuration item, which can be defined on the org."
+        )
+    logger.info(
+        f"Sending start webhook for {release} to {settings.METACI_RELEASE_WEBHOOK_URL}"
+    )
+    implementation_step_id = release.implementation_steps.get(
+        plan__role=role
+    ).external_id
+    payload = {
+        "implementation_step_id": f"{implementation_step_id}",
+    }
+    token = jwt_session()
+    response = requests.post(
+        f"http://0.0.0.0:8001/implementation_step_id/{implementation_step_id}/start/",
```

Comment:
This should use settings.METACI_RELEASE_WEBHOOK_URL instead of a hardcoded local host/port